### PR TITLE
Add param for filtering productTypes

### PR DESCRIPTION
**Pull request by junajan**
#### Summary
This PR adds a possibility to export only selected productTypes - as requested in this issue #11 

- Tests
    - [x] Unit
    - [X] Integration
    - [ ] Acceptance
- [X] Documentation

### CODE REVIEW

**Comment by junajan** [comment]
@hisabimbola - unit test added.

File: src/product-type-export.js [hunk 1]
```diff
@@ -246,11 +247,21 @@ export default class ProductTypeImport {
     })
   }
 
+  addFilterCondition (client) {
+    const keys = this.config.productTypeFilter.replace(/,/g, '","')
+    return client.where(`key IN ("${keys}")`)
```
Comment (hisabimbola) [inline src/product-type-export.js]: I am not sure we need to pass the `client` object to this method.

Let's make the method generate the keys and return it.

File: src/product-type-export.js [hunk 2]
```diff
@@ -246,11 +247,21 @@ export default class ProductTypeImport {
...
+  }
+
   downloadProductTypes (file) {
```
Comment (hisabimbola) [inline src/product-type-export.js]: Can we add a unit test for this method??

**Comment by junajan** [comment]
Closing in favor of https://github.com/sphereio/sphere-product-type-export/pull/15

**Pull request closed without merging**